lib/TextProcessing/FreePromptProvider.php

```diff
@@ -44,9 +46,11 @@ public function process(string $prompt): string {
 			throw new RunTimeException('OpenAI/LocalAI request failed: ' . $e->getMessage());
 		}
 		if (count($completion) > 0) {
+			$endTime = time();
+			$this->openAiAPIService->updateExpTextProcessingTime($endTime - $startTime);
 			return array_pop($completion);
 		}
-
+		
```

Comment:
trailing spaces

Reply:
I wondered why cs:fix didn't want to remove those. I'll push a fix!